### PR TITLE
LLM service bug 27

### PR DESCRIPTION
**Pull request by lukmanr**
With updates to unit tests for LLM Service.  Unit tests are still not passing in LLM Service but I will follow up with another PR for that.

### CODE REVIEW

**Comment by lavjain** [comment]
@lukmanr My fixes for .pylintrc are exactly the same in https://github.com/GoogleCloudPlatform/core-solution-services/pull/50/files LOL

**Review by lavjain — APPROVED** [review]
LGTM

File: .github/workflows/unit_test_linter_llm_service.yaml
Comment (jonchenn) [inline .github/workflows/unit_test_linter_llm_service.yaml:1]: This is not needed with the latest update on unit_test yaml

File: .pylintrc
Comment (jonchenn) [inline .pylintrc:1]: We use https://github.com/google/styleguide/blob/gh-pages/pylintrc, can you update based on that file?

**Review by jonchenn — CHANGES_REQUESTED** [review]
The latest unit test GHA already covers all microservices.

File: .github/workflows/unit_test_linter_llm_service.yaml
Comment (lukmanr) [reply .github/workflows/unit_test_linter_llm_service.yaml:1]: I think it's better to have separate unit test / linter action files. Let's discuss offline.

Comment (lukmanr) [reply .github/workflows/unit_test_linter_llm_service.yaml:1]: I pushed an update to remove LLM service from the unified unit test GHA for the purposes of this PR.

File: .pylintrc
Comment (jonchenn) [reply .pylintrc:1]: Let's get this PR merged, we can fix this issue later.

**Pull request merged**